index.html

```diff
@@ -870,6 +929,55 @@ <h2>Examples</h2>
         "pageSize": 2,
         "totalPages": 10
     }
+}
+              </pre></td>
+          </tr>
+          </tbody>
+      </table>
+      <br/>
+      <table id="AtomVenueForVendorVenueExample">
+          <tbody>
+          <tr>
+              <td>URL</td>
+              <td>http://api-beta.atomtickets.com/partner/v1/venues/byVendorVenueId/0001</td>
```

Comment:
Since external vendor Ids are not unique, I think we need to specify the vendor, right?

So to guarantee you're pulling the right value, we'll need Vendor's ID and Vendor's Venue ID.

Reply:
that's not how the internal api works and i don't want to make changes for that.  this is just exposing what partners were already using

Reply:
also, do you expect us to expose our internal vendor ids to the vendors? i don't think that's something we should expect them to keep track of